GPU/Common/SplineCommon.cpp

```diff
@@ -15,6 +15,13 @@
 // Official git repository and contact information can be found at
 // https://github.com/hrydgard/ppsspp and http://www.ppsspp.org/.
 
+/*
+#define WIN32_LEAN_AND_MEAN
+#include <Windows.h>
```

Comment:
`#include "Common/CommonWindows.h"` will do all these things (and use NOMINMAX.)

-[Unknown]